ur_description/urdf/ur.common.xacro

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://wiki.ros.org/xacro">
+<!--
+  Author: Ludovic Delval
+  Contributor: Felix Messmer
+-->
+  <xacro:macro name="cylinder_inertial" params="radius length mass *origin">
+    <inertial>
+      <mass value="${mass}" />
+      <xacro:insert_block name="origin" />
+      <inertia ixx="${0.0833333 * mass * (3 * radius * radius + length * length)}" ixy="0.0" ixz="0.0"
+        iyy="${0.0833333 * mass * (3 * radius * radius + length * length)}" iyz="0.0"
+        izz="${0.5 * mass * radius * radius}" />
+    </inertial>
+  </xacro:macro>
+
+  <xacro:macro name="read_file" params="joints_limits_parameters_file physical_parameters_file visual_parameters_file">
```

Comment:
Maybe I would change the name to something more meaningful like `read_model_data`, as `read_file` sounds like it is just reading one file (which was I think the original idea) but does not give much context.